client/src/reducers/hanabiReducer.js

```diff
@@ -0,0 +1,25 @@
+import { initialDeck } from '../lib/initialDeck';
+
+export default function hanabiReducer(state={}, action) {
```

Comment:
we don't have to give this a name if we are just going to export it. We will just git it a name in the rootReducer
```javascript
export default (state = {}, action) => { }
```